### DISCUSSION TITLE
milvus v2.x grafana监控面板中没有QPS和index创建和query查询耗时？

### DISCUSSION THREAD

**Discussion by xiaojuan8** [milvus-io/milvus#16046] [Q&A and General discussion]
hi，想问下如何在milvus_dashboard.json显示的grafana监控上添加QPS和耗时的监控面板？
我使用https://raw.githubusercontent.com/milvus-io/milvus/master/deployments/monitor/grafana/milvus-dashboard.json 导入grafana后，监控中没有milvus集群的QPS/QPM或者耗时等数据，我非常需要这些监控数据来评估是否能将milvus的实时查询服务接入我们推荐系统的召回服务，请帮助一下我，谢谢！

**Comment by xiaofan-luan** [comment]
Hi xiaojuan，我们在2.0.2里加入了大量指标项，包括你关注的QPS，TPS，latency，也包括耗时直方图。
可以等2.0.2发布之后看看是否足够满足你生产的需求，如果不满足欢迎持续开issue或者贡献。

**Reply by xiaojuan8** [reply]
👌，谢谢哈，开发计划(https://wiki.lfaidata.foundation/display/MIL/Milvus+2.X+Roadmap+and+Time+schedule)来看3.10会release v2.0.2， 是已经发布了嘛？ 需要更新milvus部署镜像嘛？
